### PR TITLE
Enable CentOS support for existing linux artifacts

### PR DESCRIPTION
**Pull request by ryanvog**
Use feature detection to determine package manager

### CODE REVIEW

File: Artifacts/linux-deploy-app-from-storage/deploy_from_azure_container_v1.sh [hunk 1]
```diff
@@ -1,21 +1,49 @@
...
-sudo rm -d -f -r $2
+if [ -f /usr/bin/apt ] ; then
+    echo "Using APT package manager"
 
-# download the container into the destination folder
-sudo node ./download_azure_container.js $1 $2 $3
-LAST_RESULT=$?
+    apt-get -y update
+    
+    curl -sL https://deb.nodesource.com/setup_4.x | sudo -E bash -
```
Comment (drwill-ms) [inline Artifacts/linux-deploy-app-from-storage/deploy_from_azure_container_v1.sh]: Should the trailing dash be here?

File: Artifacts/linux-install-docker-container/linux_install_docker_container.sh [hunk 2]
```diff
@@ -10,8 +12,20 @@
 #
 ###
 
-sudo apt-get -y update
-sudo apt-get -y install docker.io
+if [ -f /usr/bin/apt ] ; then
+    echo "Using APT package manager"
+
+    apt-get -y update
+    apt-get -y install docker.io
```
Comment (drwill-ms) [inline Artifacts/linux-install-docker-container/linux_install_docker_container.sh]: no need to start and enable docker like done below for the yum install path?

File: Artifacts/linux-swapfile/Artifactfile.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
         "size": {
             "type": "string",
             "displayName": "Size",
-            "description": "The size of the swap file in bytes.  Suffixes k, m, or g correspond to KiB, MiB, or GiB. Example: 5g = 5GiB, 100m = 100MiB",
+            "description": "The size of the swap file in bytes.  Suffixes k, m, or g correspond to KiB, MiB, or GiB. Exmaple: 5g = 5GiB, 100m = 100MiB",
```
Comment (drwill-ms) [inline Artifacts/linux-swapfile/Artifactfile.json]: Back to ExMaple, huh?

**Comment by drwill-ms** [comment]
"Example" misspelled again, otherwise just questions which I doubt matter.

File: Artifacts/linux-install-docker-container/linux_install_docker_container.sh (hunk 2 above)
Comment (ryanvog) [inline Artifacts/linux-install-docker-container/linux_install_docker_container.sh]: docker.io starts the service and registers it to start on boot, whereas docker-engine does not.  docker.io is not available via Yum.

File: Artifacts/linux-swapfile/Artifactfile.json (hunk 3 above)
Comment (ryanvog) [inline Artifacts/linux-swapfile/Artifactfile.json]: Fixed

File: Artifacts/linux-deploy-app-from-storage/deploy_from_azure_container_v1.sh (hunk 1 above)
Comment (ryanvog) [inline Artifacts/linux-deploy-app-from-storage/deploy_from_azure_container_v1.sh]: The dash at the end signals the end of options passed to the bash shell.

**Pull request merged**